configs/pie-components-config/package.json

```diff
@@ -5,10 +5,14 @@
   "type": "module",
   "author": "Just Eat Takeaway.com - Design System Team",
   "description": "Shared configuration files for PIE components",
+  "bin": {
+    "playwright-lit-setup": "./scripts/playwright-lit-setup.js"
+  },
   "files": [
     "custom-elements-manifest.config.js",
     "vite.config.js",
     "tsconfig.json",
+    "playwright/**",
```

Comment:
Would this also include any cache folder?

Reply:
Addressed this with a new .npmignore file 👍🏼